css/style.css

```diff
@@ -220,14 +225,18 @@ main {
   border-radius: 2px;
 }
 
-.toggle-inner {
+input[type="checkbox"] {
+  display: none;
```

Comment:
My understanding of `display:none` versus `opacity:0` (though I can't find an official resource that states unequivocally):
- `display: none` means that the element in question will not appear on the page at all (although you can still interact with it through the DOM). When I inspect the HTML for the page, the `<input>` element is present, though flagged as having a `display:none`. Doesn't this mean you still get the accessibility benefits? Also importantly: there will be no space allocated for it between the other elements.
- `opacity: 0` means that unlike `display: none`, space is allocated for it on the page. I don't want space to be allocated for it on the page, so wouldn't I want `display:none` here? If I use opacity, then I'd have to add a number of other styles to prevent the checkbox from taking up physical space on the page:
```css
input[type="checkbox"] {
  /*display: none;*/
  opacity: 0;
  height: 0;
  width: 0;
  margin: 0;
  border: 0;
}
```
And for some reason, the element still has a border around it; even though inspect tells me that the `border:0` rule is being enforced.